### ISSUE TITLE
this site does not change the fingerprint

### ISSUE THREAD

**Issue by rdbox** [cowlicks/privacypossum#114]
downloaded the latest version and decided to test, for some reason this site does not change the print ```https://panopticlick.eff.org/results```

can I do something like that?
And another question is how many percent of sites using your plug-in see us?
When is the privacypossum plug-in included that the sites see? unique user or with a distorted print

P.S. At me such thought, and probably to make not lock namely a random print with 
```usage (nodeJS local server) + application for google```
And all js parameters are fully generated.
In other words, the fingerprint2 script is used to protect and generate random values

**Comment by cowlicks** [comment]
Hello @rdbox , thank you for the question. I answered a similar question in a firefox review ([here](https://addons.mozilla.org/en-US/firefox/addon/privacy-possum/reviews/)). Basically:

TL;DR Panopticlick uses a homerolled assortment of tracking code that is impractical for commercial tracking.

I'll go into a little detail about Panopticlick to explain more. Panopticlick uses a deployment of the open source fingerprinting tool Fingerprintjs2, along with their own unique fingerprinting code.

I added some debug code and visited Panopticlick I see Privacy Possum detects the page accessing 12 API's that are marked for watching for fingerprinting. Except this is split over 3 different scripts:
https://panopticlick.eff.org/static/fp2.js
https://panopticlick.eff.org/static/fetch_whorls.js
https://panopticlick.eff.org/static/deployJava.js

Privacy possum watches for fingerprinting on *per script basis*, this is a reasonable assumption because, normally a websites tracking code is bundled into one place, so that the tracking info can be easily aggregated and used. I'm not aware of a real deployment where tracking is split up like this. It is practical for panopticlick because they want to present information about your tracking independently, and manage the code to do that in a more practical way.

For a demonstration of the fingerprinting detection code, I usually point folks to:
http://valve.github.io/fingerprintjs2/

I think it is worth considering cases like Panopticlick because it can be used to evade PP's novel detection. But I have not seen a case like this in the wild.

If you have any further questions, feel free to reopen this issue. Or create a new one.
